jcommune-model/src/main/java/org/jtalks/jcommune/model/entity/ObjectsFactory.java

```diff
@@ -155,6 +155,10 @@ public static Topic getDefaultTopic() {
         return new Topic(getDefaultUser(), "title", "Discussion");
     }
 
+    public static TopicDraft getDefaultTopicDraft() {
+        return new TopicDraft(getDefaultUser(), "topic title", "Discussion");
```

Comment:
Instead of hardcoding values it's better to generate them randomly: `RandomStringUtils.random(15)`. I know it's not used in JC yet, but we need to start moving towards this.
PS: please update the version of commons-lang to 3.x.x.